### PR TITLE
New Chess Titles Cheat Sheet

### PR DESCRIPTION
**Pull request by ArjunPanickssery**
## Description of new Instant Answer, or changes

It provides a cheat sheet for FIDE and several national chess titles.

## Related Issues and Discussions

N/A

## People to notify

N/A

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/chesstitles

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by pjhampton** [comment]
Hey @ArjunPanickssery. Welcome. 

Please update this with our [PR template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/.github/PULL_REQUEST_TEMPLATE.md). Ping me and I can reopen it.

**Comment by ArjunPanickssery** [comment]
@pjhampton I updated it.

**Comment by pjhampton** [comment]
@ArjunPanickssery reopened.

**Comment by ArjunPanickssery** [comment]
@sahildua2305 Should I submit a new pull request after using the JSON Validator?

**Comment by kirkins** [comment]
@ArjunPanickssery don't do that. Just push to your existing branch and it will update here.

**Comment by ArjunPanickssery** [comment]
@kirkins Thanks. I think I did it correctly just now.

**Comment by ArjunPanickssery** [comment]
@sahildua2305 @kirkins What does this line mean?

 daxtheduck had a problem deploying to beta.duckduckgo.com 39 minutes ago [Error]

**Comment by kirkins** [comment]
@ArjunPanickssery I think the travis.cli setup is broken until they pull https://github.com/duckduckgo/zeroclickinfo-goodies/pull/4373. 

So we won't know if the code actually passed or not until then.

**Comment by pjhampton** [comment]
Travis is now fixed. Please rebase and push your work to get the updated build information 👍

**Comment by ArjunPanickssery** [comment]
@pjhampton I think I rebased it correctly, but I'm not sure.

File: share/goodie/cheat_sheets/json/chesstitles.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{
+    "id": "chesstitles",
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/chesstitles.json]: Additionally, `id` field for a cheat sheet should be suffixed by `_cheat_sheet` which means for this case the `id` should be `chesstitles_cheat_sheet` or even better `chess_titles_cheat_sheet`.

**Comment by ArjunPanickssery** [comment]
@sahildua2305 I'll change the id and just create a new pull request. Is it a problem if the id in the JSON file is not the same as the one at https://duck.co/ia/view/chesstitles ?

**Comment by sahildua2305** [comment]
@ArjunPanickssery yes. Your ID should match at both the places (IA page as well as json field).

**Comment by sahildua2305** [comment]
@ArjunPanickssery additionally, your cheatsheets file name should be your ID except the '_cheat_sheet'.

**Comment by ArjunPanickssery** [comment]
@sahildua2305 I don't see a way to change the ID on the IA page.

**Comment by sahildua2305** [comment]
Maybe you can create a new IA?

**Comment by ArjunPanickssery** [comment]
I created a new IA at https://duck.co/ia/view/chess_titles_cheat_sheet and a new pull request at https://github.com/duckduckgo/zeroclickinfo-goodies/pull/4390.

It includes all of the changes by @sahildua2305 and @kirkins.

**Pull request closed without merging**